cumulusci/tasks/metadata/package.py

```diff
@@ -444,3 +580,62 @@ def _run_task(self):
         package_xml = self.package_xml()
         with open(self.options.get("output", output), mode="w", encoding="utf-8") as f:
             f.write(package_xml)
+
+
+class RemoveSourceComponents(object):
```

Comment:
```suggestion
class RemoveSourceComponents:
```

This is a Python 2 idiom that's no longer needed in Python 3.